from core.exceptions.CustomException import CustomException

class RouterNodeException(CustomException):
   """ Raised when there is an issue with router nodes """
   pass